db/db.go

```diff
@@ -22,13 +22,18 @@ func KMToDBPath(dir string, km *km.KeyManager) (string, error) {
 }
 
 // open a database
-func Open(path string, mode os.FileMode, options *bolt.Options) (*bolt.DB, error) {
+func Open(path, userPath string, mode os.FileMode, options *bolt.Options) (*bolt.DB, error) {
 
 	migrations := []migration.Migration{}
 
-	// migrate the database
-	err := migration.Migrate(path, migrations)
-	if err != nil {
+	// check if production database exist
+	if _, err := os.Stat(path); err == nil {
```

Comment:
if would always go with the usual ways: 
```go
_, err := os.Stat(path);
if err != nil {
   return err
}

...
```

Reply:
I just realized that we don't need to return an error because we are checking on this point if there is the need to a migration or not, but that condition doesn't lead to any error. It's just an state that will happen once. Fixed